packages/itwin/tree-widget/src/test/trees/models-tree/ModelsTreeFiltering.test.ts

```diff
@@ -577,8 +577,8 @@ describe("Models tree", () => {
           return { rootSubject, model1, model2, category1, category2, element11, element12, element21, element22 };
         },
         (x) => [
-          [x.rootSubject, x.model1, x.category1, x.element11],
-          [x.rootSubject, x.model2, x.category2, x.element22],
+          [x.rootSubject, { ...x.model1, className: "BisCore.GeometricModel3d" }, x.category1, { ...x.element11, className: "BisCore.GeometricElement3d" }],
```

Comment:
what do you think about adding a couple of functions like this:
```ts
const modelKey = (source: InstanceKey) => ({ className: "BisCore.GeometricModel3d", id: source.id });
const elementKey = (source: InstanceKey) => ({ className: "BisCore.GeometricElement3d", id: source.id });
```

and then using them when definining the paths?

Reply:
Addded two functions:
```ts
const adjustedModelKey = (source: InstanceKey) => ({ className: "BisCore.GeometricModel3d", id: source.id });
const adjustedElementKey = (source: InstanceKey) => ({ className: "BisCore.GeometricElement3d", id: source.id });
```